src/listener/Heading.php

```diff
@@ -27,10 +33,18 @@ public function process(Line $line)
 
     /**
      * {@inheritDoc}
+     * 
+     * @since 1.2.0 Added exception
+     * @throws Exception for unknown heading levels
      */
     public function render(\nadar\quill\Lexer $lexer)
     {
         foreach ($this->picks() as $pick) {
+            if (!in_array((int)$pick->heading, $this->levels)) {
```

Comment:
I think we should not type cast $pick->heading value `(int)`. Assume to very curious example of someone add is custom $levels, they must not be int values by definition.

Reply:
Sure. I was mainly unsure whether `$pick->heading` is always a string or int. And in theory there's a lot of strings (very probably not dangerous, but still), that loosely compare to `1`. Before I used the `$strict=true` to prevent that, but that won't work depending on what the type of `$pick->heading` is.